x-pack/filebeat/modules.d/zeek.yml.disabled

```diff
@@ -46,6 +46,8 @@
     enabled: true
   rfb:
     enabled: true
+  signature:
```

Comment:
You shouldn't be modifying these files directly. Rather, you can update the module configuration somewhere around here:

https://github.com/elastic/beats/blob/35e2e99e2df4fc8fe948d3dc167d42125da6934a/x-pack/filebeat/module/zeek/_meta/config.yml#L44-L47

and then run:

```
mage fmt update
```